tests/data/almanack/repo_setup/create_repo.py

```diff
@@ -144,3 +144,48 @@ def create_repositories(base_path: pathlib.Path) -> None:
     for repo_name in ["3_file_repo", "1_file_repo"]:
         repo_path = base_path / repo_name
         commit_changes(repo_path, "Commit with added lines of code")
+
+
+def create_community_health_repository(base_path: pathlib.Path) -> str:
```

Comment:
I like this approach (creating a repo and then testing this way), but is there a reason to not just apply the check test (for the includes files) directly on the almanack repo? Maybe this is circular logic however and more trouble than it's worth

Reply:
Good idea, thanks for bringing this up! I think we should add this and keep the existing isolated test data. I wondered while developing this how expectations could diverge or implement patterns that don't apply to this project (maybe that's an anti-pattern?). As a result of this comment I also added a CODE_OF_CONDUCT file which was missing but linked from the CONTRIBUTING documentation.